web/src/pages/Resolver/Parameters/Court.tsx

```diff
@@ -49,28 +50,56 @@ const AlertMessageContainer = styled.div`
   margin-top: 24px;
 `;
 
+const StyledDropdownSelect = styled(DropdownSelect)`
+  width: 84vw;
+  margin-top: 24px;
+  ${landscapeStyle(
+    () => css`
+      width: ${responsiveSize(442, 700, 900)};
+    `
+  )}
+`;
+
 const Court: React.FC = () => {
   const { disputeData, setDisputeData } = useNewDisputeContext();
   const { data } = useCourtTree();
   const items = useMemo(() => !isUndefined(data?.court) && [rootCourtToItems(data.court)], [data]);
 
-  const handleWrite = (courtId: string) => {
-    setDisputeData({ ...disputeData, courtId: courtId });
+  const disputeKitOptions = useMemo(() => {
+    if (isKlerosUniversity()) return [{ text: "Classic Dispute Kit", value: 1 }];
+    if (isKlerosNeo()) return [{ text: "Classic Dispute Kit", value: 1 }];
+    if (isTestnetDeployment()) return [{ text: "Classic Dispute Kit", value: 1 }];
+    const options = [{ text: "Classic Dispute Kit", value: 1 }];
+    if (disputeData.courtId === "1") options.push({ text: "Shutter Dispute Kit", value: 2 });
+    return options;
+  }, [disputeData.courtId]);
```

Comment:
I think it would be better if we get the information on which disputeKits are available from the subgraph 🤔
This way we wouldn't need to change depending on deployment, as the subgraph for each deployment already has the available disputeKits.
So:
- Dropdown disabled until the user selects a court
- Use as options the supported disputeKits for that court
![image](https://github.com/user-attachments/assets/272ac4ab-190d-4eba-9dd1-3bf5487b9bac)